internal/cronsumer.go

```diff
@@ -15,12 +15,18 @@ type kafkaCronsumer struct {
 
 	consumeFn func(message kafka.Message) error
 
+	metric          *CronsumerMetric
 	maxRetry        int
 	deadLetterTopic string
 
 	cfg *kafka.Config
 }
 
+type CronsumerMetric struct {
```

Comment:
We can extract metrics and all related things in a go file, for example, named as metrics.go